simulators/swan/v41.31/Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM ubuntu:22.04 as test_env
```

Comment:
Does it make sense to use such an old ubuntu?

@hpenedones should we start migrating to a new version? (or potentially to Alpine?)